x-pack/test/fleet_api_integration/apis/epm/install_custom.ts

```diff
@@ -92,11 +93,42 @@ export default function (providerContext: FtrProviderContext) {
         type: PACKAGES_SAVED_OBJECT_TYPE,
         id: INTEGRATION_NAME,
       });
-
       expect(installation.attributes.name).to.be(INTEGRATION_NAME);
       expect(installation.attributes.version).to.be(INTEGRATION_VERSION);
       expect(installation.attributes.install_source).to.be('custom');
       expect(installation.attributes.install_status).to.be('installed');
     });
+
+    it('Throws an error when there is a naming collision', async () => {
```

Comment:
Could we add an integration test to cover the case where the collision is detected with a not installed registry package?